include/frnetlib/WebSocket.h

```diff
@@ -42,10 +42,12 @@ namespace fr
          *
          * @param address The address of the socket to connect to
          * @param port The port of the socket to connect to
+         * @param path The path of the resource
+         * @param ws_protocole The Websocket-Protocol, nullptr if NA
          * @param timeout The number of seconds to wait before timing the connection attempt out. Pass {} for default.
          * @return A Socket::Status indicating the status of the operation (Success on success, an error type on failure).
          */
-        Socket::Status connect(const std::string &address, const std::string &port, std::chrono::seconds timeout) override
+        Socket::Status connect(const std::string &address, const std::string &port, const char* path,  const char* ws_protocole, std::chrono::seconds timeout)
```

Comment:
I think `const char* ws_protocole` should be `const std::vector<std::string> &ws_protocols`, for consistencies' sake.